it/controllers/incomeSources/cease/IncomeSourceEndDateControllerISpec.scala

```diff
@@ -202,5 +318,59 @@ class IncomeSourceEndDateControllerISpec extends ComponentSpecBase {
       }
     }
   }
+
+  s"calling GET $dateForeignPropertyShowChangeUrl" should {
+    "render the Date Foreign Property Ceased Page" when {
+      "User is authorised" in {
+        Given("I wiremock stub a successful Income Source Details response with Foreign property")
+        enable(IncomeSources)
+        IncomeTaxViewChangeStub.stubGetIncomeSourceDetailsResponse(testMtditid)(OK, foreignPropertyOnlyResponse)
+
+        When(s"I call GET $dateForeignPropertyShowChangeUrl")
+        val result = IncomeTaxViewChangeFrontend.getForeignPropertyEndDate
```

Comment:
the request should be made to `change-foreign-property-end-date`

Reply:
changed